pom.xml

```diff
@@ -90,13 +90,6 @@
           </excludes>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Does this mean checkstyle is turned off by default? Wouldn't it be better to have it default to no violations with console output?